app/main/posts/modify/post-editor.html

```diff
@@ -33,15 +33,15 @@ <h1 class="mode-context-title">{{post.form.name}}</h1>
 
           <div class="form-sheet">
               <div class="post-band" style="background-color: {{form.color}}"></div>
-              <div class="form-field title form-field-adapt init"
+              <div class="form-field title init required"
```

Comment:
Ideally just make these fixes direct to develop I think. For now though we can just go ahead with this PR